package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spectacle",
-  "version": "6.2.0",
+  "version": "7.0.0-beta.1",
```

Comment:
Do we want to do this straight from `main-7` for `beta|next` npm publish? (Don't mind here or in main-7, just thought I'd ask).

Reply:
Ah yes, good point!